### ISSUE TITLE
Error while running example.py

### ISSUE THREAD

**Issue by paulbosmans** [JakubAndrysek/pySpaceMouse#3]
Hi, I am receiving the following error on a raspi installation, any idea where it come from ? 

```
Traceback (most recent call last):
  File "/home/user/git/example.py", line 8, in <module>
    state=pyspacemouse.read()
  File "/home/user/.local/lib/python3.9/site-packages/pyspacemouse/pyspacemouse.py", line 586, in read
    return _active_device.read()
  File "/home/user/.local/lib/python3.9/site-packages/pyspacemouse/pyspacemouse.py", line 209, in read
    self.process(ret)
  File "/home/user/.local/lib/python3.9/site-packages/pyspacemouse/pyspacemouse.py", line 241, in process
    flip * to_int16(data[b1], data[b2]) / float(self.axis_scale)
IndexError: bytearray index out of range

```

**Comment by JakubAndrysek** [comment]
Which version of space mouse have you got?

**Comment by JakubAndrysek** [comment]
Have you installed hidapi?
`sudo apt-get install libhidapi-dev`

**Comment by paulbosmans** [comment]
Thank you for your answer. I have the spacemouse wireless, and hidapi is installed. The code seems to work until I touch the mouse and I get this error

**Comment by JakubAndrysek** [comment]
I don't know where the error could be.

Have you solved it?

**Comment by emil302s** [comment]
from what I could see sometimes the modules try to read byte 8 in the data but it only goes to 7
I don't know why it happens but the way I fixed it was to check that b1 and b2 never went over the length of the data and if they did pass.

**Comment by JakubAndrysek** [comment]
Hi, thank you for your research.
Could you create a pull request with your fix? I will look at it.

Thanks

**Comment by JakubAndrysek** [comment]
Fixed #5
